docs/_getting-started/dev-mode.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Dev mode
+---
+
+Geth has a development mode which sets up a single node Ethereum test network with a number of options optimized for developing on local machines. You enable it with the `--dev` argument.
+
+Starting geth in dev mode does the following:
+
+-   Initializes the data directory with a testing genesis block
+-   Sets max peers to 0
+-   Turns off discovery by other nodes
+-   Sets the gas price to 0
+-   Uses the ethash consensus engine with clique headers to allows blocks to be mined as fast as your CPU and memory can handle
+-   Prevents the consensus (proof of work) difficulty from ever increasing
```

Comment:
Nah, strike this. It doesn't use PoW, but there is a notion of `difficulty` which is used by Clique to figure out which chain is canonical, but that's a bit too much detail probably

Reply:
```suggestion
```